WooCommerce/WooCommerceTests/ViewRelated/Orders/Order Creation/EditableOrderViewModelTests.swift

```diff
@@ -624,15 +622,7 @@ final class EditableOrderViewModelTests: XCTestCase {
         // When
         productSelectorViewModel.changeSelectionStateForProduct(with: product.productID)
         productSelectorViewModel.completeMultipleSelection()
-        let testFeeLine = OrderFeeLine(feeID: 0,
-                                       name: "Fee",
-                                       taxClass: "",
-                                       taxStatus: .none,
-                                       total: "10",
-                                       totalTax: "",
-                                       taxes: [],
-                                       attributes: [])
-        viewModel.saveFeeLine(testFeeLine)
+        viewModel.saveFeeLine("10")
```

Comment:
This is great, tests are much more readable now that logic is moved to `saveFeeLine` internals  🚀